osu.Game/Screens/Play/Player.cs

```diff
@@ -89,6 +89,8 @@ public class Player : ScreenWithBeatmapBackground
 
         public BreakOverlay BreakOverlay;
 
+        public IBindable<bool> IsBreakTime => breakTracker?.IsBreakTime;
```

Comment:
Should probably add xmldoc, and it should never be null. If there's a reason `breakTracker` can be null then initialise this bindable locally and bind to the `breakTracker` one in load/`LoadComplete`